ros_ethercat_model/include/ros_ethercat_model/hardware_interface.hpp

```diff
@@ -95,12 +96,20 @@ class ActuatorCommand
 public:
   ActuatorCommand() :
     enable_(0),
-    effort_(0)
+    effort_(0),
+    effort_command_type_(EffortCommandType::UNSPECIFIED)
   {
   }
 
   bool enable_;  //!< Enable this actuator
-  double effort_;  //!< Force to apply (in Nm)
+  double effort_;  //!< Effor to apply, unit depends on effort_command_type_ field
+  EffortCommandType effort_command_type_;  //!< Type of effort in effort_ field
+
+  void set_effort_command(double effort, EffortCommandType effort_command_type)
```

Comment:
Do we need a set method? None of the other fields have one.
Maybe it's better to keep the style uniform?

Reply:
Idea was for all new code to use setter method because both fields should be set at the same time. Happy to remove it.